helpers/guard.js

```diff
@@ -0,0 +1,19 @@
+const passport = require('passport');
+const { HttpCode } = require('../configuration/const');
+
+const guard = (req, res, next) => {
```

Comment:
Cały ten plik to do middleware folderu powinien iść i nazywać się najlepieje auth.js i tam samo w pliku czyli const auth. (sugestia)